manifests/p/Proton/ProtonMail/0.9.3/Proton.ProtonMail.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Proton.ProtonMail
+PackageVersion: 0.9.3
+PackageLocale: en-US
+Publisher: Proton AG
+PackageName: Proton official desktop application for Proton Mail and Proton Calendar

```

```suggestion
PackageName: Proton Mail Beta
```
This should match control panel